src/Avalonia.Controls/Remote/Server/RemoteServerTopLevelImpl.cs

```diff
@@ -301,8 +310,7 @@ protected void RenderIfNeeded()
                     return;
 
             }
-            if (ClientSize.Width < 1 || ClientSize.Height < 1)
```

Comment:
What would happen if those are actually < 1? Or even < 0?

Reply:
If these are 0, then rendering is skipped over at line [276](https://github.com/AvaloniaUI/Avalonia/pull/2307/files/1f25265170c95ceb1f7ff2ba2b4ed35b565c84e9#diff-95da73f791f195c28fa00ea4a0696163R276).

Regarding <0 is that possible?